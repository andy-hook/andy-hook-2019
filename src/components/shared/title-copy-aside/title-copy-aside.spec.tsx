import React from "react"
import TitleCopyAside from "./title-copy-aside"
import { renderWithTheme } from "@test-utils"

describe("<TitleCopyAside />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <TitleCopyAside title="This is a title">
        This is child content
      </TitleCopyAside>
    )
    expect(tree).toMatchSnapshot()
  })
})
