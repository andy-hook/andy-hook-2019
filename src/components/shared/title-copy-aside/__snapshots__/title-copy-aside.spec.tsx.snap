// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleCopyAside /> renders correctly 1`] = `
<div
  className="sc-bdVaJa kZAgjJ"
>
  <h2
    className="sc-bwzfXH gyarJK"
  >
    This is a title
  </h2>
  <p
    className="sc-htpNat kMdGYa"
  >
    This is child content
  </p>
</div>
`;
